src/pages/AssignServerRoles.js

```diff
@@ -495,11 +495,8 @@ class AssignServerRoles extends BaseWizardPage {
     let expTime = new Date(now);
     expTime.setMinutes(now.getMinutes() + minutes);
 
-    let retOp = {path: '/', expires: expTime, secure: true, sameSite: true};
-    //allow http for development
-    if(getAppConfig('dev')) {
-      retOp.secure = false;
-    }
+    // if later decide to always run installer with https, add secure: true
+    let retOp = {path: '/', expires: expTime, sameSite: true};
```

Comment:
Shouldn't the presence 'secure' option in the cookie be controlled by the "Secured" checkbox that the user sets on the form rather than basing it on an option in our config file?

Reply:
@GarySmith  secure option in cookie session is not related to Secured option for session cookie. Secured option in configuration is more of for we use a certificate (ssl) to communicate with oneview or suse manager. Session key's secure option indicates that we have to run installer with https. I added secure option for session key because I thought we will eventually run installer with https after had the security session with security team. Since at this point we don't have plan to run https, think remove it for now until we decide to run installer with https.

Reply:
@GarySmith Added a SUSE Jira item as a reminder if we decide to run https as default down the road. Thanks for the comments.